xapian-bindings/python3/docs/introduction.rst

```diff
@@ -13,6 +14,12 @@ them, you'll need to add this to your code::
 
   import xapian
 
+It also includes `xapianletor` bindings module - to use
+them, you'll need to configure xapian-bindings with --enable-letor=yes
```

Comment:
Just `--enable-letor` should work and seems clearer.

It's also helpful to mark up command options so they stand out visually when converted to HTML, PDF, etc - double backticks render their content as a fixed width font, which it like `<code>` or `<tt>` in HTML: ` ``--enable-letor`` `

Reply:
fixed in a fixup commit.